zipline/pipeline/loaders/events.py

```diff
@@ -1,15 +1,21 @@
-import abc
+import numpy as np
 import pandas as pd
-from six import iteritems
-from toolz import merge
+
+from toolz import groupby, merge
 
 from .base import PipelineLoader
 from .frame import DataFrameLoader
-from .utils import previous_event_frame, next_event_frame
-from zipline.pipeline.common import TS_FIELD_NAME
-from zipline.utils.numpy_utils import NaTD
-
-WRONG_COLS_ERROR = "Expected columns {expected_columns} for sid {sid} but " \
+from zipline.pipeline.common import (
+    EVENT_DATE_FIELD_NAME,
+    SID_FIELD_NAME,
+    TS_FIELD_NAME,
+)
+from zipline.pipeline.loaders.utils import (
+    next_event_indexer,
+    previous_event_indexer,
+)
+
+WRONG_COLS_ERROR = "Expected columns {expected_columns} for but " \
```

Comment:
We can probably get rid of all the errors defined here other than the `WRONG_COLS_ERROR`.